charts/enterprise-kyverno-operator/templates/_helpers.tpl

```diff
@@ -154,3 +154,12 @@ Create secret to access container registry
 {{- define "enterprise-kyverno.policysetsStr" -}}
 {{- range (include "enterprise-kyverno.enabledPolicysets" . | split ",") }}{{(print . " ") }} {{- end }}
 {{- end -}}
+
+{{- define "kyverno.excludedNamespaces" -}}
+{{- $defaultNamespaces := list "kube-system" "nirmata" "nirmata-system" -}}
+{{- if eq .Values.cloudPlatform "openshift" -}}
+{{- $defaultNamespaces = append $defaultNamespaces "openshift-*" }}
+{{- end -}}
+{{- $excludedNamespaces := concat $defaultNamespaces .Values.kyverno.excludedNamespacesForWebhook -}}
```

Comment:
ExcludeNamespaces by default has the above 3 namespaces and openshift (plus openshift cluster). But user can set exclude it to empty in which case, those will not be excluded. Even for openshift.